pkg/repository/olap.go

```diff
@@ -92,12 +92,12 @@ type WorkflowRunData struct {
 
 type OLAPEventRepository interface {
 	ReadTaskRun(ctx context.Context, taskExternalId string) (*timescalev2.V2TasksOlap, error)
-	ReadWorkflowRun(ctx context.Context, tenantId, workflowRunExternalId pgtype.UUID) (*WorkflowRunData, []TaskMetadata, error)
+	ReadWorkflowRun(ctx context.Context, workflowRunExternalId pgtype.UUID) (*WorkflowRunData, []TaskMetadata, error)
```

Comment:
Let's add a custom return type which contains both `*WorkflowRunData` and `[]TaskMetadata`?

Reply:
works for me!